test/test_sync.cpp

```diff
@@ -127,6 +127,7 @@ ClientHistory& get_history(DBRef db)
     return get_replication(db).get_history();
 }
 
+#if !REALM_MOBILE // the server is not implemented on devices
```

Comment:
There are no functional changes to the code in this file, just moving tests under the !REALM_MOBILE flag if they used the server fixtures.